added disjoint smart graph documentation to release notes new features (#428)
arangodb/docs: 3.7/release-notes-new-features37.md

SatelliteGraphs are only available in the Enterprise Edition and the
[ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).

> Disjoint SmartGraphs

Reviewer: ```suggestion
DisjointSmartGraphs
```

--------------------

SmartGraphs have been extended with a new option `isDisjoint`.

> A Disjoint SmartGraph prohibits edges connecting different SmartGraph

Reviewer: ```suggestion
A DisjointSmartGraph prohibits edges connecting different SmartGraph
```

components. If your graph doesn't need edges between vertices with different
SmartGraph attribute values, then you should enable this option. This topology
restriction allows the query optimizer to improve traversal execution times.

> Disjoint SmartGraphs are only available in the Enterprise Edition and the

Reviewer: ```suggestion
DisjointSmartGraphs are only available in the Enterprise Edition and the
```

SatelliteGraphs are only available in the Enterprise Edition and the
[ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).

> DisjointSmartGraphs
> ---------------

Reviewer 2: ```suggestion
Disjoint SmartGraphs
--------------------
```

SmartGraphs have been extended to a new subtype, called **DisjointSmartGraphs**.

> A DisjointSmartGraph prohibids connected edges between different SmartGraph

Reviewer 3: ```suggestion
A DisjointSmartGraph prohibits connected edges between different SmartGraph
```

> SmartGraphs have been extended to a new subtype, called **DisjointSmartGraphs**.
> A DisjointSmartGraph prohibits connected edges between different SmartGraph
> components. In case the graph schema is able to be represented without the need
> of connected SmartGraph components, a DisjointSmartGraph should be used as this
> knowledge can be used by the internal optimizer to gain even better query
> traversal execution times.
> 
> 
> DisjointSmartGraphs are only available in the Enterprise Edition and the
> [ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).

Reviewer 2: ```suggestion
SmartGraphs have been extended with a new option `isDisjoint`.
A Disjoint SmartGraph prohibits edges connecting different SmartGraph
components. If your graph doesn't need edges between vertices with different
SmartGraph attribute values, then you should enable this option. This topology
restriction allows the query optimizer to improve traversal execution times.

Disjoint SmartGraphs are only available in the Enterprise Edition and the
[ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).
```
